src/Symfony/Bridge/Monolog/Handler/ConsoleHandler.php

```diff
@@ -162,6 +164,9 @@ protected function write(array $record)
      */
     protected function getDefaultFormatter()
     {
+        if (!class_exists(CliDumper::class)) {
```

Comment:
I did not find a way to test this (other than change the namespace of the test to match the one of the class but I'm don't think that would be ok with you).